ledger/participant-integration-api/src/main/scala/platform/store/FlywayMigrations.scala

```diff
@@ -21,6 +21,10 @@ private[platform] class FlywayMigrations(jdbcUrl: String)(implicit loggingContex
 
   private val dbType = DbType.jdbcType(jdbcUrl)
 
+  // The last stable version.
+  // Across all database types, experimental migrations must have a higher version number.
+  private val lastStableSchemaVersion = "49"
```

Comment:
please mark it up with `TODO append-only` stating this feature is temporary and will be removed in the cleanup step